doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1343,43 +1343,49 @@ A subscription is represented by a JSON object with the following fields:
   If this field is not provided at subscription creation time, new subscriptions are created with
   the `active` status, which can be changed by clients afterwards. For expired subscriptions this
   attribute is set to `expired` value (no matter if the client updates it to `active`/`inactive`).
++ `throttling`: Minimal period of time in seconds which must elapse between two consecutive notifications. It
+  is optional.
 
 A `subject` contains the following subfields:
 
 + `entities`: A list of objects, each one composed of the following subfields (`id`/`idPattern` or `type` must be present):
     + `id` or `idPattern`: Id or pattern of the affected entities. Both cannot be used at the same time, but at least one of them must be set.
     + `type`: Type of the affected entities (optional).
 + `condition`: Condition that will trigger the notification. It can have two optional properties:
-    + `attributes`: array of attribute names
+    + `attrs`: array of attribute names
     + `expression`: an expression composed of `q`, `georel`, `geometry` and `coords` (see "List entities" operation above
    about this field).
 
 A `notification` object contains the following subfields:
 
-+ `attributes`: List of attributes to be included in the notification message. If not specified (or empty), all attributes are included
++ `attrs`: List of attributes to be included in the notification message. If not specified (or empty), all attributes are included
   in the notification. It also defines the order in which attributes will appear in notifications when `attrsFormat` `value` is used
   (see "Notification messages" section).
-+ `callback` : URL pointing to the service which will be invoked when a notification is generated. A NGSIv2
-  compliant server must support `http` URL schema, other schemas (e.g. schemas for web sockets) could also be supported.
-+ `headers` (optional): a key-map of HTTP headers that will be included in the notification messages. Note that this cannot be used
-  to override headers that make the notification inconsistent (e.g. `Content-Type: application/json`).
-+ `query` (optional): a key-map of URL query parameters that will be included in the notification message.
++ `http`: Information about the HTTP transport.
```

Comment:
Thanks for the suggestion! I was trying to find a more descriptive sentence this morning and, due to lack of ideas, I ended in a very poor one ;)